src/modules/orders/services/export.service.ts

```
@@ -0,0 +1,136 @@
+          }
+          resolve(filePath);
+        });
+        ws.on('error', (error) => {
+          clearTimeout(writeTimeout);
+          reject(error);
+        });
+      } catch (error) {
+        clearTimeout(writeTimeout);
+        reject(error);
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)